Subject: domcaps: Fix check for uncached security features
From: Cole Robinson crobinso@redhat.com Tue Nov 12 14:15:56 2019 -0500
Date: Tue Nov 12 14:17:21 2019 -0500:
Git: d934d6f266e1731a5b881bbb9ae4cd60acd7887e

We need to check against None, which is the initial value, otherwise
a host with none of the security features present will repeatedly poll
libvirt baseline APIs

Signed-off-by: Cole Robinson <crobinso@redhat.com>

diff --git a/virtinst/domcapabilities.py b/virtinst/domcapabilities.py
index ea0ffd1f..34991ed7 100644
--- a/virtinst/domcapabilities.py
+++ b/virtinst/domcapabilities.py
@@ -296,7 +296,7 @@ class DomainCapabilities(XMLBuilder):
                 'virt-ssbd',
                 'md-clear']
 
-        if self._features:
+        if self._features is not None:
             return self._features
 
         self._features = []
